Docs/reference/content/reference/driver_core/sdamevents.md

```diff
@@ -0,0 +1,126 @@
++++
+date = "2019-04-17T21:21:00Z"
+draft = false
+title = "SDAM Events"
+[menu.main]
+  parent = "Eventing"
+  identifier = "SdamEvents"
+  weight = 90
+  pre = "<i class='fa'></i>"
++++
+
+## SDAM Events
+
+Server Discovery and Monitoring (SDAM) is the process by which the driver discovers and monitors the set of servers that it is connected to. In
+the case of a standalone configuration the driver will only monitor a single server. In the case of a replica set configuration the driver
+will monitor each member of the replica set (primary, secondaries, etc...). In the case of a sharded configuration the driver will  
+monitor the set of shard routers (mongos instances) that it is connected to.
+
+As the driver monitors the health and state of each server in a configuration it raises a number of events that report what it is finding and
```

Comment:
Minor nit: Is a comma needed? 
"As the driver monitors the health and state of each server in a **configuration, it** raises a number of events that report what it is finding..."